src/AcceptanceTests/Routing/When_sending_to_specific_namespace.cs

```diff
@@ -13,56 +11,38 @@ namespace NServiceBus.Azure.Transports.WindowsAzureServiceBus.AcceptanceTests.Ro
 
     public class When_using_multiple_namespaces : NServiceBusAcceptanceTest
     {
+        static string connectionString = connectionString = EnvironmentHelper.GetEnvironmentVariable("AzureServiceBusTransport.ConnectionString");
```

Comment:
We sprinkle these these environment variable names like there's no tomorrow 😄 
While there's no issue for now, we'll have to consolidate it in another PR for issue #626.